STYLE.md

```diff
@@ -0,0 +1,12 @@
+#Style Guild
```

Comment:
If you folks want to use a markdown file for your styleguide, totally fine, but go research markdown syntax and utilize the full suite of styling abilities. You can do things like:
-  `code blocks` 
- lists
- [links](https://google.com)
- [] check boxes
- etc...